### PR TITLE
VMware: add new module vmware_vmkernel_facts

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME

lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py
test/integration/targets/vmware_vmkernel_facts/aliases
test/integration/targets/vmware_vmkernel_facts/tasks/main.yml

##### ANSIBLE VERSION

```
2.5devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
ready_for_review

File: lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                        ipv4_address=vmk.spec.ip.ipAddress,
+                        ipv4_subnet_mask=vmk.spec.ip.subnetMask,
+                        dhcp=vmk.spec.ip.dhcp,
+                        mac=vmk.spec.mac,
+                        mtu=vmk.spec.mtu,
+                        stack=vmk.spec.netStackInstanceKey,
+                        enable_vsan=True if vmk.device in self.service_type_vmks[host.name]['vsan'] else False,
+                        enable_vmotion=True if vmk.device in self.service_type_vmks[host.name]['vmotion'] else False,
+                        enable_management=True if vmk.device in self.service_type_vmks[host.name]['management'] else False,
+                        enable_ft=True if vmk.device in self.service_type_vmks[host.name]['faultToleranceLogging'] else False,
```
Comment (tchernomax) [inline lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py]: `True if foo in bar else False` → `(foo in bar)`
it's already a boolean

**Review by tchernomax — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py (hunk 1 above)
Comment (tchernomax) [reply lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py]: ping @Akasurde

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py]: OK.

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_vmkernel_facts.py]: When you think too much about code, these kinda of things will happen. :)

**Review by tchernomax — APPROVED** [review]
LGTM

**Comment by Akasurde** [comment]
@tchernomax Thanks for review.

**Comment by misteryiz** [comment]
ship it. my test cases passed without any issues :)

**Comment by Akasurde** [comment]
@misteryiz Thanks for testing.

**Pull request merged**